### ISSUE TITLE
0.8.0 (in pipy) doesn't support 3.0

### ISSUE THREAD

**Issue by Grazfather** [jbardin/scp.py#47]
Looks like py3 support was added. Could you update the release version?

**Comment by remram44** [comment]
#41 is a pretty serious issue so it might make sense to wait for #43 to be reviewed and merged.

**Comment by jbardin** [comment]
yes, I need to work on that. I'll get #43 merged asap

**Comment by Grazfather** [comment]
Thanks!

**Comment by kimgr** [comment]
Could you update the version number in master so that it doesn't conflict with the current PyPI release?

I've gone rogue and installed scp.py directly from GitHub with:

    $ pip install https://github.com/jbardin/scp.py/archive/master.tar.gz

but now I'm in the process of building a wheel for an internal repo, and it'd be nice if I didn't have to lie and say it was v0.8.0 (the PyPI version).

Thanks!

**Comment by jbardin** [comment]
Hi @kimgr,

The latest master is what I'm planning to push for 0.9.0. It had quite a few changes, can you give that a try for me?

PS. If you're not aware, you can use pip to pin a dependency at a specific git commit or tag, so you're not stuck worrying about master breaking, e.g.

    pip install git+git://github.com/jbardin/scp.py.git@563d78a

or in requirements.txt

  scp=git+git://github.com/jbardin/scp.py.git@563d78a

**Comment by kimgr** [comment]
The latest HEAD works great for me, thanks!

Also thank you for the pip tips, I didn't know about the git-specific syntax. It doesn't work for me though, because I republish packages in an internal repository. We find that it makes pip install faster and more dependable, but we also run something like 3-4000 builds/week in our lab, so any glitch hurts us faster than most people. Also, we keep a trail of historical versions to be able to reproduce old releases, etc.

So in order to publish internally, I need a version number that I can correlate with a specific public version. The GitHub URL versioning scheme doesn't transfer between repos easily.

Anyway, thank you for the pure-Python SCP support, I really appreciate it!

**Comment by remram44** [comment]
#32 and #33 relate to specific error scenarios and can probably be postponed. Or is 0.9 waiting on them?

**Comment by jbardin** [comment]
yeah, those will have to wait. Fixing them will be a little more intrusive, but at least now I can test those conditions more reliably.

**Comment by remram44** [comment]
:ship: ship it :shipit:

**Comment by Grazfather** [comment]
:boom: Thanks guys!

**Comment by kimgr** [comment]
Lovely, thanks!
